scripts/gen_help_html.lua

```diff
@@ -80,6 +80,9 @@ local exclude_invalid_urls = {
   ['http://wiki.services.openoffice.org/wiki/Dictionaries'] = 'spell.txt',
   ['http://www.adapower.com'] = 'ft_ada.txt',
   ['http://www.jclark.com/'] = 'quickfix.txt',
+  ['http://oldblog.antirez.com/post/redis-and-scripting.html'] = 'faq.txt',
+  ['http://seriot.ch/parsing_json.php'] = 'faq.txt',
```

Comment:
https seems to work for this one: https://seriot.ch/projects/parsing_json.html

Reply:
I didn't write url checkeer

Reply:
All of them work

Reply:
Probably worth a pass across the docs to switch to https where possible (separate PR).